src/pages/view-test-result/cat-a-mod1/components/speed-card/__tests__/speed-card.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { async, TestBed } from '@angular/core/testing';
+import { IonicModule, Config } from 'ionic-angular';
+import { ConfigMock } from 'ionic-mocks';
+import { configureTestSuite } from 'ng-bullet';
+import { SpeedCardComponent } from '../speed-card';
+
+describe('SpeedCardComponent', () => {
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        SpeedCardComponent,
+      ],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: Config, useFactory: () => ConfigMock.instance() },
+      ],
+    });
+  });
+
+  beforeEach(async(() => {
+  }));
+
+  describe('DOM', () => {
+
```

Comment:
Empty test file

Reply:
writing one now.

Reply:
removed file and added DOM test into its parent